src/clarity-angular/data/datagrid/datagrid-cell.ts

```diff
@@ -56,7 +37,43 @@ export class DatagridCell {
      *
      * @type {string}
      */
-    public id: string;
+    private _id: string;
+
+    set id(value: string) {
+        this._id = value;
+        this.mapHideableColumn(this._id);
+    }
+
+    private _hiddenStateSubscription: Subscription;
```

Comment:
I believe our conventions would want this to be `private hiddenStateSubscription: Subscription;` without the `_` since its not a get/set property like `id` above.

Reply:
Thanks @gnomeontherun. I've updated as requested.